### PR TITLE
Drop levels below 3

### PR DESCRIPTION
**Pull request by hrw**
Kind of implements #141 

Commit messages need work.

### CODE REVIEW

File: test_pool/io_virt/test_i007.c [hunk 1]
```diff
@@ -39,11 +39,9 @@ static void payload()
 
   while (num_smmu--) {
       if (val_smmu_get_info(SMMU_CTRL_ARCH_MAJOR_REV, num_smmu) == 2) {
-          if (g_sbsa_level > 3) {
```
Comment (gowthamsiddarthd) [inline test_pool/io_virt/test_i007.c:42]: For level equal to 3, SMMU v3 support is not mandatory. It is required only from level 4 and above. So, this change is not required.

File: val/src/avs_pe.c [hunk 2]
```diff
@@ -66,22 +66,17 @@ val_pe_execute_tests(uint32_t level, uint32_t num_pe)
...
-      status |= c017_entry(num_pe);
-  }
+  status |= c016_entry(num_pe);
+  status |= c017_entry(num_pe);
 
   if (level > 5) {
       status |= c018_entry(num_pe);
   }
 
-  if (level > 3) {
```
Comment (gowthamsiddarthd) [inline val/src/avs_pe.c:79]: These tests are applicable for levels 4 and above. Not required to be executed for level 3. So, this change is not required.

**Review by gowthamsiddarthd — COMMENTED** [review]
Hi @hrw,

As mentioned, the commit message needs work. 
The rest of the changes looks fine.

Regards,
Gowtham

**Comment by hrw** [comment]
Thanks for review @gowthamsiddarthd.

Review comments resolved, commit messages rewritten.

File: test_pool/pe/test_c001.c [hunk 3]
```diff
@@ -31,22 +31,10 @@ payload()
...
-          return;
-      }
-  } else {
-      if (num_of_pe > MAX_NUM_PE_LEVEL2) {
-          val_set_status(index, RESULT_FAIL(g_sbsa_level, TEST_NUM, 01));
-          val_print(AVS_PRINT_ERR, "Number of PE is %d \n", num_of_pe);
-          return;
-      }
-
+  if (num_of_pe > MAX_NUM_PE_LEVEL2) {
```
Comment (chetan-rathore) [inline test_pool/pe/test_c001.c:34]: As there is no LEVEL2 from 5.0 onwards, the macro can be renamed to #define MAX_NUM_OF_PE

**Comment by gowthamsiddarthd** [comment]
Hi @chetan-rathore,

I will take up this change and also change the copyright year of the modified files.

Regards,
Gowtham

**Review by gowthamsiddarthd — APPROVED** [review]
LGTM

**Pull request merged**